posthog/temporal/batch_exports/bigquery_batch_export.py

```diff
@@ -195,13 +201,16 @@ def bigquery_default_fields() -> list[BatchExportField]:
 
 
 @activity.defn
-async def insert_into_bigquery_activity(inputs: BigQueryInsertInputs) -> int:
+async def insert_into_bigquery_activity(inputs: BigQueryInsertInputs) -> RecordsCompleted:
     """Activity streams data from ClickHouse to BigQuery."""
     logger = await bind_temporal_worker_logger(team_id=inputs.team_id, destination="BigQuery")
     logger.info(
-        "Exporting batch %s - %s",
+        "Batch exporting range %s - %s to BigQuery: %s.%s.%s",
```

Comment:
Also, I've standardized these log lines at the beginning of each batch export. I think they look nicer: even if they are not that useful, when I am a user I like to see stuff being printed, makes it seem stuff is working fine. Maybe it's me having too much fun :shrug: